platform/migration/platform-migration-commands/src/main/java/org/codice/ddf/migration/commands/ExportCommand.java

```diff
@@ -25,8 +27,11 @@
  * are exported to the directory specified in the implementation of the service
  * (see {@link ConfigurationFileDirectory} for an example).
  */
-@Command(scope = PlatformCommands.NAMESPACE, name = "config-export", description = "Exports configurations")
-public class ExportCommand extends PlatformCommands {
+@Command(scope = MigrationCommands.NAMESPACE, name = "export", description = "Exports the entire system in order "
+    + "to prepare for a migration to a new system. Exported items may include configurations, files, catalog contents, etc.. "
+    + "The export command calls the Migration Service to execute the export of all settings. This in turn delegates "
```

Comment:
Sounds a little too technical and design dependent. Should we write something like `The export command delegates to all registered Migratable services to export bundle specific configuration and data.` or something similar?